src/json.c

```diff
@@ -626,6 +627,12 @@ lyjson_value(struct lyjson_ctx *jsonctx)
         LY_CHECK_RET(lyjson_array(jsonctx));
 
     } else if (*jsonctx->in->current == '{') {
+        (jsonctx->depth)++;
+        if (jsonctx->depth > LY_MAX_BLOCK_DEPTH) {
+            LOGERR(jsonctx->ctx, LY_EINT,
```

Comment:
Not an internal error, they are only those that should never occur under normal circumstances (and are hence independent of any user input). This is probably `LY_EINVAL`.